packages/devtools_app/test/memory/diff/widgets/class_filter_test.dart

```diff
@@ -13,20 +14,42 @@ import 'package:flutter_test/flutter_test.dart';
 import '../../../matchers/matchers.dart';
 import '../../../scenes/memory/diff_snapshot.dart';
 
+final _customFilter =
+    ClassFilter(filterType: ClassFilterType.only, except: '', only: '');
+
+class _FilterTest {
```

Comment:
This can probably be changed to an enum:

```dart
enum FilterTestType {
  diff,
  single,
}
```

And then instead of creating `_tests` below, you can just do `for (test in FilterTestType.values) {...}`, using `test.name` to get `'diff'` or `'single'`.

Reply:
Good idea. However, _FilterTest turned out to be convenient place to generate paths for tests, so I will stay with it.

Reply:
This can still be an enum:

```dart
enum FilterTestType {
  diff,
  single;

  String get sceneGolden => '../../../goldens/memory_diff_snapshot_scene_$name.png';
  // ...
}

```